schema/2025-03-26/schema.ts

```diff
@@ -802,6 +811,120 @@ export interface Tool {
   annotations?: ToolAnnotations;
 }
 
+
+/**
+ * Sent from the client to request a list of augmentations the server has.
+ */
+export interface ListAugmentationsRequest extends PaginatedRequest {
+  method: "augmentation/list";
+}
+
+/**
+* The server's response to an augmentation/list request from the client.
+*/
+export interface ListAugmentationsResult extends PaginatedResult {
+  augmentations: Augmentation[];
+}
+
+/**
+* An optional notification from the server to the client, informing that the list
+* of augmentations it offers has changed. This may be issued by servers without
+* any previous subscription.
+*/
+export interface AugmentationListChangedNotification extends Notification {
+  method: "notifications/augmentation/list_changed";
+}
+
+/**
+ * Request to augment context with additional information.
+ */
+export interface AugmentRequest {
+  method: "augmentation/augment";
+  params: {
+      /**
+       * The name of the augmentation to perform.
+       */
+      name: string;
+      /**
+       * The context to augment. This can include text, images, or other resources.
+       */
+      context: (TextContent | ImageContent | AudioContent | EmbeddedResource)[];
```

Comment:
THese should probably use the [just-merged](https://github.com/modelcontextprotocol/modelcontextprotocol/pull/603) `ContentBlock` type.